### PR TITLE
jenkins: update all image tags to 2019-03-08

### PR DESCRIPTION
**Pull request by lamping7**
Docker hub is still building, but lets begin testing the new tag https://github.com/PX4/containers/releases/tag/2019-03-08

I'll re-trigger Jenkins later after Docker hub gets done, then probably grind it few times.

### CODE REVIEW

**Comment by lamping7** [comment]
It appears that appveyor annoyingly fails if you push a new commit while it is building the last...

**Comment by dagar** [comment]
> It appears that appveyor annoyingly fails if you push a new commit while it is building the last...

That's okay, this otherwise looks good to go?

**Comment by lamping7** [comment]
Yes. I was planing on letting the slaves die off and run one more. After that I'll clear the grind commits and move it from draft to PR stage. Give me an hour or so.

File: Jenkinsfile [hunk 1]
```diff
@@ -75,7 +77,7 @@ pipeline {
 
         stage('Style check') {
           agent {
-            docker { image 'px4io/px4-dev-base:2019-02-09' }
+            docker { image 'px4io/px4-dev-base-bionic:2019-03-08' }
```
Comment (dagar) [inline Jenkinsfile]: bionic?

Comment (dagar) [reply Jenkinsfile]: Nevermind, forgot what we're doing now.

Comment (lamping7) [reply Jenkinsfile]: Yup... moving from 16 -> 18

**Comment by lamping7** [comment]
It's worth saying for others that I killed px4-dev-base and px4-dev-ros in docker hub. They sill exist but won't update any longer. Because we're in the midst of transition the images have been re-banded to reflect to the systems they run as defined in the [readme](https://github.com/PX4/containers/blob/master/docker/px4-dev/README.md)

**Pull request merged**